### PR TITLE
Updated unit tests

### PR DESCRIPTION
**Pull request by stephanvierkant**
I joined this library as a maintainer a while ago and I struggle to handle the Pull Requests because I my knowledge of all internals of this library is a bit limited. Therefor some extra unit tests are very helpful, to prevent regression. 

This is a first step to improve tests for this library. In the feature, I'd like to see updated unit tests as a requirement to every PR.

What I've done so far:
* Upgraded Symfony requirement to supported versions: 3.4 (LTS) or ^4.1. Other versions don't get security updates to it's important to upgrade as soon as possible.
* Fixed some PHPUnit deprecations
* Added some columns to `PostDatatable`
* Added PHP 7.2 and 7.3 to Travis, remove PHP 7.1. since we don't support that version anymore.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,13 +18,19 @@
         "issues": "https://github.com/stwe/DatatablesBundle/issues"
     },
     "require": {
-        "php": ">=7.1",
-        "symfony/framework-bundle": "^3.0|^4.0",
+        "php": ">=7.2",
```
Comment (Seb33300) [inline composer.json]: Why not keeping PHP 7.1 as a minimum requirement?
I think we should follow the minimum version required by Symfony.

File: composer.json [hunk 2]
```diff
@@ -18,13 +18,19 @@
...
+        "symfony/framework-bundle": "^3.4|^4.1",
```
Comment (Seb33300) [inline composer.json]: Same. Why upgrading Symfony minimum version requirement?

Comment (Seb33300) [reply composer.json]: BTW if this project can work on 3.4, it should be working on 4.0

File: composer.json (hunk 1 above)
Comment (stephanvierkant) [reply composer.json]: I'm sorry, requiring 7.2 is a bit strict. I will change this!

File: composer.json (hunk 2 above)
Comment (stephanvierkant) [reply composer.json]: Support for 4.0 has been dropped: https://symfony.com/roadmap/4.0

I would work on 4.0 in most cases, but not necessarily. I've had cases were 2.8 (LTS) received an imported bugfix for a new PHP release, while a non-LTS 3.x version didn't get that bugfix. 

Supporting an older major Symfony/PHP version (Symfony 3.4, PHP 5.6 when it was still supported) makes sense in most cases, upgrading to a minor release (PHP 7.0 -> PHP 7.1, Symfony 4.0 -> Symfony 4.1) isn't that hard.

**Pull request merged**